### ISSUE TITLE
Cannot convert OmniSR to ONNX

### ISSUE THREAD

**Issue by RunDevelopment** [chaiNNer-org/chaiNNer#1816]
**Information:**

- Chainner version: nightly
- OS: Win10

**Description**
As pointed out by @hooke007, OmniSR models cannot be converted to ONNX. I can reproduce this too.

**Logs**

This is what was logged to console: [console.log](https://github.com/chaiNNer-org/chaiNNer/files/11541190/console.log)

**Comment by joeyballentine** [comment]
I don't think this is something i can fix so we should probably just give a good user facing message on how this can't be converted to onnx

**Comment by RunDevelopment** [comment]
I thought that #1805 was specifically for OmniSR, was it not?

**Comment by joeyballentine** [comment]
Oh yeah true wtf

**Comment by joeyballentine** [comment]
I'll take a look later today

**Comment by hooke007** [comment]
Does it help with this issue? https://github.com/Phhofm/models/blob/main/2xHFA2kAVCOmniSR/onnx/2xHFA2kAVCOmniSR2onnx.py

**Comment by joeyballentine** [comment]
That does it the same way we do it... So it doesn't make sense why it would work there and not here. I'll check it out again tomorrow if I remember to

**Comment by hooke007** [comment]
The latest version works but data of `shape` is lost in the onnx.
